tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -635,11 +624,34 @@ function main() {
 
 		case "$action" in
 		install)
+			if [[ "$runtime" =~ ^(k3s|k3s-agent|rke2-agent|rke2-server)$ ]]; then
+			       if [ ! -f "$containerd_conf_tmpl_file" ] && [ -f "$containerd_conf_file" ]; then
+				       cp "$containerd_conf_file" "$containerd_conf_tmpl_file"
+			       fi
+			       # Only set the containerd_conf_file to its new value after
+			       # copying the file to the template location
+			       containerd_conf_file="${containerd_conf_tmpl_file}"
+			       containerd_conf_file_backup="${containerd_conf_tmpl_file}.bak"
+			elif [[ "$runtime" =~ ^(k0s-worker|k0s-controller)$ ]]; then
+			       touch "$containerd_conf_file"
+			elif [[ "$runtime" == "containerd" ]]; then
```

Comment:
This was the fallback in the existing code. What's the rationale to make it an `elif` ?

Reply:
This was the fallback when cri-o was being checked.  As there's no CRI-O check at this point, this if we make it the fallback it'd apply for both `containerd` and `crio` as the runtime, which we don't want to happen.

Reply:
Of course I missed that :grin: . Thanks for the clarification !